tests/reference_jobfile_hdd

```diff
@@ -1,6 +1,6 @@
 timeout 5s
 hdd 1
 hdd-bytes 100M
-hdd-opts direct, fsync, wr-rnd
+hdd-opts direct,fsync,wr-rnd
```

Comment:
This looks like a likely user error in the making.  Is there something that we can do so that it doesn't matter if the user puts them in there or not?  (E.g., can we make our code remove them if they are present?)

Reply:
As above, note that `reference_jobfile_hdd` is an example stress-ng input file, which the test uses to compare it's generated input file against. There is nothing to protect the user against here as it is the job of the plugin to take the schema-defined input and convert it into the stress-ng input file format.

Reply:
Note that the spaces were there when I originally put this together because I thought they were neede/accepted, but later tests revealed that they need to explicitly not be there.